Enabling bare metal

evedon opened the pull request:
Information on how to enable bare metal is missing from this page (although it can be found here: https://os.mbed.com/docs/mbed-os/v5.14/tutorials/migrating-to-mbed-os-5.html)

evedon committed:
Enabling bare metal

evedon commented:
@AnotherButler Please review asap. This information is now difficult to find in the docs and should be corrected.

Commit:
Update baremetal_example.md

Replace duplicated content with link to the specific section with this content. This prevents errors due to duplication.

AnotherButler commented:
@evedon I've replaced the duplication with a link to the specific section. If you think this is still too difficult to find, please let me know.

evedon commented:
> @evedon I've replaced the duplication with a link to the specific section. If you think this is still too difficult to find, please let me know.

I think it should be the other way round. This section in the docs is about bare metal so it should describe how to enable it. The other page is a porting doc.

AnotherButler commented:
Good point. I switch it, so the other page points to this one, instead.

Commit:
Update baremetal_example.md

Add bare metal enabling instructions back, so we can link the other location to this one.
